modules/ROOT/pages/app-dashboards.adoc

```diff
@@ -58,10 +58,18 @@ Built-in dashboards include various pages to view different types of charts.
 * Total inbound requests
 * Average response time inbound
 * Average response time by endpoint
+** Avg: green if response time is under 300ms; yellow if response time is greater than 300ms and lower than 600ms; red if is greater than 600ms
```

Comment:
```suggestion
** Avg: Green if response time is under 300 ms; yellow if response time is greater than 300 ms and lower than 600 ms; red if is greater than 600 ms.
```

Reply:
In this and all the subsequent ms values: add the space but, more important, how do we account for the actual number in between "under"/"lower than" and "greater than"? 

For example, in the above, what if the value is actually 300 ms on the dot, or 600 ms?

GLOBAL in this topic.

Reply:
And standardize to "less than" and "greater than." GLOBAL

Reply:
```suggestion
** Average: Green if response time is less than 300ms; yellow if response time is greater than 300ms and less than 600ms; red if is greater than 600ms
```

Reply:
I'll ask dev about greater/less than or equal to. The chances of being exact are probably low given the UoM, but it's worth mentioning.